doc/user-guide/hierarchical-data.rst

```diff
@@ -74,17 +72,17 @@ We now have a small family tree
     homer
 
 where we can see how these individual Simpson family members are related to one another.
-The nodes representing Bart and Lisa are now connected - we can confirm their sibling rivalry by examining the :py:class:`~DataTree.siblings` property:
+The nodes representing Bart and Lisa are now connected - we can confirm their sibling rivalry by examining the :py:class:`~xarray.DataTree.siblings` property:
 
 .. ipython:: python
 
-    list(bart.siblings)
+    list(homer["Bart"].siblings)
 
-But oops, we forgot Homer's third daughter, Maggie! Let's add her by updating Homer's :py:class:`~DataTree.children` property to include her:
+But oops, we forgot Homer's third daughter, Maggie! Let's add her by updating Homer's :py:class:`~xarray.DataTree.children` property to include her:
 
 .. ipython:: python
 
-    maggie = DataTree(name="Maggie")
+    maggie = xr.DataTree(name="Maggie")
     homer.children = {"Bart": bart, "Lisa": lisa, "Maggie": maggie}
```

Comment:
After https://github.com/pydata/xarray/pull/9477 is merged, let's switch to:
```suggestion
    homer.children["Maggie"] = maggie
```

Reply:
Raised https://github.com/pydata/xarray/issues/9489 to track this specific issue, which should get fixed as part of #9477.

Reply:
Noted on #9489